### PR TITLE
Python3 doctest compatibility: PaddingAndApodization to PoldiCreatePeaksFromCell

### LINKED ISSUE #20811: Python3 doctest compatibility: PaddingAndApodization to PoldiCreatePeaksFromCell
Python3 doctest compatibility: algorithms <PaddingAndApodization to PoldiCreatePeaksFromCell>

This issue is part of the move to Python3 in the doctests, 
Algorithms covered in this issue are:
```
PaddingAndApodization-v1
Pause-v1
PawleyFit-v1
PeakIntensityVsRadius-v1
PeaksInRegion-v1
PeaksOnSurface-v1
PerformIndexOperations-v1
PhaseQuad-v1
PlotAsymmetryByLogValue-v1
PlotPeakByLogValue-v1
Plus-v1
PoissonErrors-v1
PoldiAnalyseResiduals-v1
PoldiAutoCorrelation-v5
PoldiCreatePeaksFromCell-v1
```

### PR DESCRIPTION
**Pull request by brandonhewer**
Python3 doctest compatibility: algorithms <PaddingAndApodization  to PoldiCreatePeaksFromCell>

This PR is part of refactoring effort to bring Python3 compatiblity into the doctests.
See https://github.com/mantidproject/mantid/issues/20672 for details.

The following tests were refactored to be compatible :
```
PaddingAndApodization-v1
Pause-v1
PawleyFit-v1
PeakIntensityVsRadius-v1
PeaksInRegion-v1
PeaksOnSurface-v1
PerformIndexOperations-v1
PhaseQuad-v1
PlotAsymmetryByLogValue-v1
PlotPeakByLogValue-v1
Plus-v1
PoissonErrors-v1
PoldiAnalyseResiduals-v1
PoldiAutoCorrelation-v5
PoldiCreatePeaksFromCell-v1
```

Code review.

Fixes #20811.
Internal change, no release notes

### CODE REVIEW

File: docs/source/algorithms/Plus-v1.rst [hunk 1]
```diff
@@ -83,9 +83,9 @@ Output:
     # perform the algorithm
     ws = ws1 + ws2
 
-    print "The X values are: " + str(ws.readX(0))
-    print "The Y values are: " + str(ws.readY(0))
-    print "The updated Error values are: " + str(ws.readE(0))
+    print("The X values are: " + str(ws.readX(0)))
+    print("The Y values are: " + str(ws.readY(0)))
+    print("The updated Error values are: " + str(ws.readE(0)))
```
Comment (rosswhitfield) [inline docs/source/algorithms/Plus-v1.rst]: There are more `print` statements in this file that need to be fixed.

File: docs/source/algorithms/PlotAsymmetryByLogValue-v1.rst [hunk 2]
```diff
@@ -55,8 +55,8 @@ Usage
                                 TimeMin = 0.55,
                                 TimeMax = 12.0);
 
-   print "Y values (asymmetry):", ws.readY(0)
-   print "X values (sample magn. field):", ws.readX(0)
+   print("Y values (asymmetry): {}".format(ws.readY(0)))
+   print("X values (sample magn. field): {}".format(ws.readX(0)))
```
Comment (rosswhitfield) [inline docs/source/algorithms/PlotAsymmetryByLogValue-v1.rst]: There are more `print` statements in this file that need to be fixed.

File: docs/source/algorithms/PeaksInRegion-v1.rst [hunk 3]
```diff
@@ -26,15 +26,15 @@ Usage
...
    
    # Find peaks in region when the peak is just outside the box (by radius)
    just_outside_box_table = PeaksInRegion(peaks, CoordinateFrame='HKL', PeakRadius=0.999, Extents=[1.0,2.0,-1.0,1.0,-1.0,1.0], CheckPeakExtents=True)
-   print just_outside_box_table.row(0)
+   print(just_outside_box_table.row(0))
    
    # Find peaks in region when the peak is just inside the box (by radius)
    just_intesecting_box_table = PeaksInRegion(peaks, CoordinateFrame='HKL', PeakRadius=1.00, Extents=[1.0,2.0,-1.0,1.0,-1.0,1.0], CheckPeakExtents=True)
-   print just_intesecting_box_table.row(0)
+   print(just_intesecting_box_table.row(0))
```
Comment (rosswhitfield) [inline docs/source/algorithms/PeaksInRegion-v1.rst]: These are printing a `dict` and the order is not guaranteed for python 3

File: docs/source/algorithms/PeaksOnSurface-v1.rst [hunk 4]
```diff
@@ -29,16 +29,16 @@ Usage
...
    # Peak is off the plane, and does not intesect it
    out_of_plane_offset = 1.000
    tbl = PeaksOnSurface(InputWorkspace=peaks, PeakRadius=0.999,  CoordinateFrame='HKL', Vertex1=[1.0, -1.0, out_of_plane_offset], Vertex2=[-1.0,-1.0,out_of_plane_offset], Vertex3=[-1.0, 1.0,out_of_plane_offset], Vertex4=[1.0, 1.0,out_of_plane_offset])
-   print tbl.row(0)
+   print(tbl.row(0))
    
    # Peak is off the plane, but does intesect it when radius is made larger
    tbl = PeaksOnSurface(InputWorkspace=peaks, PeakRadius=1.000,  CoordinateFrame='HKL', Vertex1=[1.0, -1.0, out_of_plane_offset], Vertex2=[-1.0,-1.0,out_of_plane_offset], Vertex3=[-1.0, 1.0,out_of_plane_offset], Vertex4=[1.0, 1.0,out_of_plane_offset])
-   print tbl.row(0)
+   print(tbl.row(0))
```
Comment (rosswhitfield) [inline docs/source/algorithms/PeaksOnSurface-v1.rst]: Again these are printing a `dict` and the order is not guaranteed for python 3

**Review by rosswhitfield — CHANGES_REQUESTED** [review]

**Comment by josephframsay** [comment]
Brandon asked me to take over these PRs as he's not here until 16th

**Review by rosswhitfield — APPROVED** [review]

**Pull request merged**